Resources/ElfStats.gd

```diff
@@ -31,6 +31,25 @@ class Stat:
 		
 	func get_value() -> float:
 		return changed_value
+		
+	func get_value_replaced_item(item) -> float:
+		var vwc = value
+		
+		for c in changers:
+			if c.item_name != item.name:
+				vwc = c.get_multiplayed_value(vwc)
```

Comment:
What is `c`? and `vwc`? maybe some better name or comment ?

Reply:
Fixed:

c - changer,